user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: Cloud_Connection_Issues
+---
+
+# Cloud Connection Issues
+This page provides solutions for issues that may arise from connectivity problems with cloud services.
+
+## Remote Access Issues
+Follow the guide below when you are encountering connection issues with dataminer.services features and your DataMiner System.
```

Comment:
This sentence doesn't say much and could be removed imo

Reply:
Going to leave it, it makes a subcategory for specific issues about Remote Access.